src/librekey/rnp_key_store.c

```diff
@@ -182,13 +184,17 @@ rnp_key_store_load_from_file(rnp_t *rnp, rnp_key_store_t *key_store, const unsig
 
             memset(&mem, 0, sizeof(mem));
 
+            if (rnp_get_debug(__FILE__)) {
+                fprintf(rnp->io->errs, "Loading G10 key from file '%s'\n", path);
```

Comment:
Well... key-store has a lot of fprints(..) for debug/errors proposal. So, I think that better switch it to `RNP_LOG_FD` after and as separated step.